packages/netlify-cms-backend-github/src/implementation.js

```diff
@@ -272,14 +284,22 @@ export default class GitHub {
     }
   }
 
-  updateUnpublishedEntryStatus(collection, slug, newStatus) {
-    return this.api.updateUnpublishedEntryStatus(collection, slug, newStatus);
+  getStatusLabels(status) {
+    return Map({
+      [status.get('DRAFT')]: Map({ name: 'draft', color: 'fad8c7' }),
+      [status.get('PENDING_REVIEW')]: Map({ name: 'pending: review', color: 'fef2c0' }),
+      [status.get('PENDING_PUBLISH')]: Map({ name: 'pending: publish', color: 'c2e0c6' }),
```

Comment:
Ideally these would be configurable, but that's a pretty complex thing to do because allowing change means handling cleanup, which is non-trivial since the CMS doesn't know what it's configuration **used** to be.

So for now, we can at least help ensure against collisions by making the names more specific:

- cms/draft
- cms/review
- cms/publish

Reply:
To further strengthen against collisions we can add the `netlify` string, thoughts  ?

Reply:
Yeah that makes sense, agreed.